project/src/ui/file_dialog/FileLoader.gd

```diff
@@ -0,0 +1,35 @@
+# A class for loading files into the system
+#
+# Usage:
+# var file_load = load("res://src/ui/file_dialog/FileLoader.gd")
+# var content = file_load.loadFile ("path to file") #get file contents
+# file_load.saveFile("path to file","content to record") #record into file
+#
+	
+
+
+# returns file contents if file exists, null otherwise
+static func loadFile(filepath: String):
+	var f = File.new()
+	if f.file_exists(filepath):
+		f.open(filepath, File.READ)
```

Comment:
you need to check if open returned success, omit the file existence check and directly open the file